docs/4p.md

```diff
@@ -0,0 +1,51 @@
+# 4P instance
+
+##  Acquisition session metadata file
+
+### How the generate acquisition session metadata template with PHIS data ?
+
+- Web app Client
+
+Once the template file has been associated to the PHIS instance infrastructure with the right type (e.g. http://www.phenome-fppn.fr/vocabulary/2017#UAV), you can generate a new file with PHIS data within.
+
+The latest file (sort by the creation date) with the wanted type is retrevied and stored on the server.
+
+- Webservice
+
+A sheet named HiddenPHIS is added to this file which is filled with the data retreived from the service `GET /acquisitionSessions/metadataFile`. You must fill the  `vectorRdfType` GET parameter (e.g. http://www.phenome-fppn.fr/vocabulary/2017#UAV) to retreive the data.
```

Comment:
it's not clear, the WS doesn't add the sheet. you should remove the first sentence. May be put it in the first paragraph

Reply:
I understand , i will modify it